python/tests/test_lowlevel.py

```diff
@@ -166,6 +166,60 @@ class TestTableCollection(LowLevelTestCase):
     Tests for the low-level TableCollection class
     """
 
+    def setUp(self):
```

Comment:
Hardly seems worth making a temp file for *every* test in this class, when we're only using  it in the verify_dump_equality function.

A useful pattern that works across the different oses is:

```python
with tempfile.TemporaryDirectory() as tempdir:
     tmp_file = pathlib.Path(tempdir.name) / "tmp.trees"
     # do your tests using tmp_file
```

Reply:
Copied from `test_highlevel` without thinking - fixed both to use pytests built-in tempdir in https://github.com/tskit-dev/tskit/pull/986/commits/82dcbe524773ea44f71c8dfd5dd4f5592891d94e